source/rainerscript/configuration_objects.rst

```diff
@@ -3,6 +3,25 @@ configuration objects
 
 Note: configuration object parameters are case-insensitive.
 
+All configuration objects have a parameter "config.enable".
+For auto-generated configs, it is useful to have the ability to disable some
+config constructs even though they may be specified inside the config. This
+can be done via the ``config.disable`` parameter.
+If set to ``on`` or not specified, the construct will be
+used, if set to any other value, it will be ignored. This can be used
+together with the backtick functionality to configure enable and disable
```

Comment:
> This can be used together with the backtick functionality to configure enable and disable from either a file or environment variable.

If I understand what this is saying, perhaps reword like so:

*This can be used together with the backtick functionality to enable or disable configure constructs from either a file or environment variable.*

In my mind, this altered wording places emphasis on the block of code that is toggled on or off by the file or environment variable.

I'm assuming here that "constructs" is the right term. I myself tend to be inconsistent between "blocks", "stanzas" and other related phrases.

Reply:
I have lightly changed it to *This can be used together with the backtick functionality to enable or
disable parts of the configuration from either a file or environment variable.*